async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Does the customer need to substitute in their own bucket name? If so, please use the SageMaker default bucket in this example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Updated to use sagemaker default bucket